diff --git a/source/cairo/loader.d b/source/cairo/loader.d
index 75e539f..6aee33a 100644
--- a/source/cairo/loader.d
+++ b/source/cairo/loader.d
@@ -34,8 +34,9 @@ CairoSupport loadCairo() {
                 "libcairo.dylib"
             ];
     } else version (Posix) {
-        const(char)[][2] libNames =
+        const(char)[][3] libNames =
             [
+                "libcairo.so.2",
                 "libcairo.so",
                 "/usr/local/lib/libcairo.so",
             ];
