diff --git a/source/cairo/loader.d b/source/cairo/loader.d
index 75e539f..598b563 100644
--- a/source/cairo/loader.d
+++ b/source/cairo/loader.d
@@ -22,7 +22,7 @@ bool isCairoLoaded() {
     return lib != invalidHandle;
 }
 
-CairoSupport loadCairo() {
+CairoSupport loadCairo() @nogc {
     version (Windows) {
         const(char)[][1] libNames =
             [
@@ -51,7 +51,7 @@ CairoSupport loadCairo() {
     return ret;
 }
 
-CairoSupport loadCairo(const(char)* libName) {
+CairoSupport loadCairo(const(char)* libName) @nogc {
     lib = load(libName);
     if (lib == invalidHandle) {
         return CairoSupport.noLibrary;
