diff --git a/modules/canvas/soba/canvas/package.d b/modules/canvas/soba/canvas/package.d
index fae289d..93182dd 100644
--- a/modules/canvas/soba/canvas/package.d
+++ b/modules/canvas/soba/canvas/package.d
@@ -42,7 +42,7 @@ bool cnvInit() {
     // TODO: Try Blend2D
 
     // Try Cairo
-    CairoSupport support = loadCairo();
+    CairoSupport support = loadCairo("libcairo.so.2");
     if (support == CairoSupport.cairo) {
         cnvBackend = SbCanvasBackend.cairo;
         return true;
diff --git a/source/soba/core/package.d b/source/soba/core/package.d
index 067eeb7..3e2ccf2 100644
--- a/source/soba/core/package.d
+++ b/source/soba/core/package.d
@@ -24,5 +24,5 @@ void sbInit() {
 
     SDL_Init(SDL_INIT_EVERYTHING);
 
-    loadCairo();
+    loadCairo("libcairo.so.2");
 }
\ No newline at end of file
